app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ErrorsController < ApplicationController
+
+  def not_found
+    render(status: 404)
+  end
+
+  def unprocessable
+    render(status: 422)
+  end
+
+  def internal_server_error
```

Comment:
Just spotted this - is this not a bit dangerous ?  If you have a 500 error because of something inside the app that happens to generate a 500 error even when this view is rendered - we would end up in a loop..  I think this is why generally these error pages are always static pages

Reply:
@garytaylor you wouldn't get in the loop. I tried. You would get just another error page but without any helpful information or formatting. But I know what you mean. I using dynamic error page so we use the translation version for them. The other way would be to have a static page for every error in 2 languages. I'm happy to talk about it,

Reply:
Im probably not the best person to talk to about it - maybe one of the devops guys - Isher or Ben might have an opinion ?